src/components/BadTextInput.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <v-text-field
+    @input="update"
```

Comment:
Kseniya says this `@` is a shorthand for `v-on-` - https://vuejs.org/v2/guide/syntax.html#v-on-Shorthand